node-api-dotnet/samples/NodeKernelSyntaxExamples/README.md

```diff
@@ -9,6 +9,23 @@ This project enables advanced interoperability between .NET and JavaScript in th
 To run the samples, first set the following environment variables referencing your
 [Azure OpenAI deployment](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/quickstart):
 
-- `OPENAI_ENDPOINT`
-- `OPENAI_DEPLOYMENT`
-- `OPENAI_KEY`
+``` bash
+# Azure OpenAI
+set AzureOpenAI__ServiceId=..
```

Comment:
I'd add a value here like this so it's clear what this represents:

`set AzureOpenAI__ServiceId=text-davinci-003`

Reply:
Going to keep this similar to the pattern here: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/KernelSyntaxExamples/README.md

`text-davinci-003` isn't a good example to use because you cannot deploy it anymore.

But I take the point and I'll add more details to the README so there is a better getting started experience.